packages/integration-tests/tests/retrospective-demo/vote.test.ts

```diff
@@ -0,0 +1,172 @@
+import {test, expect} from '@playwright/test'
+import config from '../config'
+import {goToNextPhase, skipToVotePhase} from './retrospective-demo-helpers'
+
+test.describe('retrospective-demo / vote page', () => {
+  test('it allows voting up a group', async ({page}) => {
+    await skipToVotePhase(page)
+    const voteCount = page.locator(`[data-cy="Start-group-0"] [data-cy="completed-vote-count"]`)
+    await expect(voteCount).toHaveText('0')
+    await page.locator(`[data-cy="Start-group-0"] [aria-label="Add vote"]`).click()
+    await expect(voteCount).toHaveText('1')
+  })
+
+  test('it reduces the number of votes remaining', async ({page}) => {
+    await skipToVotePhase(page)
+    const totalRemainingVotes = page.locator('div:right-of(:text("My Votes")) >> nth=0')
```

Comment:
These positional selectors are cool. It says "the 0th div to the right of the "My Votes" text".

https://playwright.dev/docs/selectors#selecting-elements-based-on-layout